repos/roslyn-tools.proj

```diff
@@ -2,27 +2,20 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
   <PropertyGroup>
-    <PackagesOutput>$(ProjectDirectory)/artifacts/$(Configuration)/packages</PackagesOutput>
-    <RepoApiImplemented>false</RepoApiImplemented>
+    <BuildCommand>$(ProjectDirectory)\build$(ShellExtension) --pack --configuration $(Configuration)</BuildCommand>
     <OrchestratedManifestBuildName>N/A</OrchestratedManifestBuildName>
+    <LatestCommit>9bb9aec2fdfd165eef3776fd3ff2f1bb9b63e46d</LatestCommit>
+    <UsesRepoToolset>true</UsesRepoToolset>
+    <BuiltToolPackageId>$(RepoToolsetPackageId)</BuiltToolPackageId>
   </PropertyGroup>
 
   <ItemGroup>
     <EnvironmentVariables Include="UsingToolMicrosoftNetCompilers=false" />
     <!-- Match the version symreader wants - this is the highest among our client repos -->
     <EnvironmentVariables Include="BUILD_BUILDNUMBER=20180215.01" />
```

Comment:
Is there a plan for keeping the BuildNumber updated?

Reply:
Hmm, I don't think it needs to be set to anything in particular anymore, since now it's auto-updated rather than matched up forcefully. I'm going to try removing this line to produce dev-versioned builds.